doc/concepts/how_to_guide.rst

```diff
@@ -1,43 +0,0 @@
-How-To Guide
-============
-
-The How-To Guides are distinctly different from tutorials in that they are directed towards a different audience.
-How-To Guides are **Problem-Oriented** and assume the reader came to this page to answer a specific question.
-These pages posses these qualities:
-
-* a series of steps
-* a focus on the goal
-* addressing a specific question
```

Comment:
you deleted this file instead of moving it, did you mean to do that?

Reply:
I deleted this deliberately. I didn't quite see a How-To Guide was a MoveIt concept so maybe it should be elsewhere.

Reply:
Can you figure out where it would fit instead of deleting it?

Reply:
Maybe create a Concepts or Background section at the top of the corresponding page in Contributing and move it there?